packages/core/addon/components/perfect-scrollbar.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *  {{#perfect-scrollbar options=(hash of options)}}
+ *    Content to scroll over.
+ *  {{/perfect-scrollbar}}
+ *
+ */
+
+import Component from '@ember/component';
+import layout from '../templates/components/perfect-scrollbar';
+import PerfectScrollbarMixin from 'navi-core/mixins/perfect-scrollbar';
+import { computed } from '@ember/object';
+
+export default Component.extend(PerfectScrollbarMixin, {
+  layout,
+
+  /**
+   * perfect scrollbar options hash.
+   *
+   * @property {Object} options
```

Comment:
Can we keep this jsdoc styles consistent with others?

Ex:
https://github.com/yahoo/navi/blob/master/packages/reports/addon/components/dimension-bulk-import.js#L30

Reply:
That styles is not agreed upon across the entire project.  After doing some research about js doc it seems like very short annotations should be put inline after the dash, while more in depth descriptions should be place on separate lines.